src/AWSMetadataUtilities.jl

```diff
@@ -85,26 +89,28 @@ function _get_service_and_version(filename::String)
 end
 
 """
-    _generate_low_level_definitions(services::AbstractArray{OrderedDict})
+    _generate_low_level_definitions(services::AbstractArray{<:AbstractDict}, repo_name::String, auth::GitHub.OAuth2)
 
 Get the low-level definitions for all AWS Services.
 
 # Arguments
-- `services::Abstract{OrderedDict}`: List of AWS Services to generate low-level definitions
+- `services::Abstract{<:AbstractDict}`: List of AWS Services to generate low-level definitions
+- `repo_name::String`: aws-sdk-js repository
+- `auth::GitHub.OAuth2`: OAuth2 credentials for authenticated requests
 
 # Returns
 - `Vector{String}`: Array of low-level service code to be written into `AWSServices.jl`
 """
-function _generate_low_level_definitions(services::AbstractArray{OrderedDict})
+function _generate_low_level_definitions(services::AbstractArray, repo_name::String, auth::GitHub.OAuth2)
     service_definitions = String[]
 
     for service in services
-        service_name = service["name"]
+        service_name = service["path"]
         @info "Generating low level wrapper for $service_name"
 
         # Get the contents of the ${Service}.normal.json file
-        request = HTTP.get(service["download_url"])
-        service_metadata = JSON.parse(String(request.body))["metadata"]
+        response = JSON.parse(String(base64decode(blob(repo_name, service["sha"]; auth=auth).content)))
+        service_metadata = response["metadata"]
```

Comment:
```suggestion
        service_blob = blob(repo_name, service["sha"]; auth=auth)
        service = JSON.parse(String(base64decode(service_blob.content)))
        service_metadata = service["metadata"]
```

line was too long, changed to match _generate_high_level_wrapper